Misc/NEWS.d/next/C API/2021-02-18-18-46-42.bpo-35134.dFpEDT.rst

```diff
@@ -0,0 +1,3 @@
+Move odictobject.h, parser_interface.h, picklebufobject.h, pydebug.h, and
+pyfpe.h into the cpython/ directory. They must not be included directly, as
+they are already included by Python.h: :ref:`Include Files <api-includes>`.
```

Comment:
Hum. If we move more header files, the risk of breaking 3rd C extensions is higher. I'm still supportive of this change, but please document it also at C API > Porting to Python 3.10:
https://docs.python.org/dev/whatsnew/3.10.html#id2

In What's New in Python 3.10, document also the other header files that you already moved.

Reply:
Updated. I assume the main concern is if they have been included directly? I've worded my note with that in mind, but let me know if I should add anything else.

I've taken a quick look at the remaining files: there are still more than 10 header files that could be moved into cpython/ wholesale, and about the same number which require some surgery. Would you support making these changes (hopefully to close bpo-35134 entirely) within 3.10?